lib/propshaft/asset.rb

```diff
@@ -32,6 +32,10 @@ def digested_path
     end
   end
 
+  def stale?(digest)
```

Comment:
Better to flip this to fresh? so you don't need to do a negative in the one place it's used.

Reply:
Oh right. I remember that controllers had `stale?` and forgot about `fresh_when` when I was choosing a name.
Done.